checker/examples/units-extension/FrequencyRelations.java

```diff
@@ -1,16 +1,22 @@
 import javax.annotation.processing.ProcessingEnvironment;
 import javax.lang.model.element.AnnotationMirror;
+import java.util.Set;
 
 import org.checkerframework.checker.units.UnitsRelations;
-import org.checkerframework.checker.units.qual.*;
+import org.checkerframework.checker.units.qual.UnknownUnits;
+import org.checkerframework.checker.units.qual.Prefix;
 import org.checkerframework.framework.type.AnnotatedTypeMirror;
 import org.checkerframework.framework.util.AnnotationBuilder;
+import org.checkerframework.javacutil.AnnotationUtils;
 
 /*>>>
 import org.checkerframework.checker.nullness.qual.Nullable;
-*/
+ */
 
-/** Relations among units of frequency. */
+/** Relations among units of frequency.
+ * Skip test below ensures the framework's automated tests does not consider this file as a test file.
```

Comment:
I again don't see why this file needs to be skipped. Should it be SupressWarnings for unwanted units warnings? Why are there units warnings?